src/index.html

```diff
@@ -17,11 +17,417 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link should point to a CSS file, not a SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Please ensure the href attribute points to a valid CSS file.